src/ir/enum_ty.rs

```diff
@@ -49,6 +54,11 @@ impl Enum {
         &self.variants
     }
 
+    /// The special attributes of enumeration
+    pub fn attributes(&self) -> &Vec<Attribute> {
```

Comment:
Agree, for the the struct layout, `alignof(T)` have enough information for most of scenes

On the other hand, sometime we need field layout beyond its type. 

For example, when parse a packet header with fixed size prefix,
```C++
struct header
{
    char proto;
    unsigned int size __attribute__ ((packed));
    unsigned char data[] __attribute__ ((aligned(8)));
} __attribute__ ((aligned));
```
We hard to generate the right padding bytes without knowledge of the field layout.

Another option seems could use [clang_Cursor_getOffsetOfField](http://clang.llvm.org/doxygen/group__CINDEX__TYPES.html#gaa7e0f0ec320c645e971168ac39aa0cab) and add a offset property to the field. (I'm not familiar with LLVM API, just found it minutes ago)